scalableminds/webknossos
# Implement new "welcome header"

The new welcome header with the empty-datasets-CTA currently looks like this:

![image](https://user-images.githubusercontent.com/2486553/106581915-b0e5da80-6543-11eb-9f4f-fa1df5f9f5a6.png)


In addition to the new header, I also removed the redundant headlines from the dashboard tab contents (as they merely repeated the tab titles).

Remaining todo:
- [x] add UI to hide the banner
- [x] persist whether the header was hidden by the user. the old banner was set via a GET parameter, which meant that the user would see this only once. should have a JSON-like property for a user to store stuff like this? @fm3 What do you think? for https://github.com/scalableminds/webknossos/issues/5022 we would need this, too. db migrations can update values within a json object, right?
- [x] clean up (e.g., remove code of old whats next banner)

### URL of deployed dev instance (used for testing):
- https://welcomeheader.webknossos.xyz

### Steps to test:
- log in on the dev instance
- check out the welcome banner in the dashboard and test its different actions
  - feel free to test the close-icon, but note that the header won't re-appear on page refresh. so, do this at the very end of testing :laughing: 
- checkout the empty dataset table with the overhauled CTA cards
  - they will only appear if the dataset list is empty. I hardcoded this for now to simplify testing.

### Issues:
- fixes #5023
- fixes #5020

------
